### DISCUSSION TITLE
美國工程師職場上常用的術語 Part1

### DISCUSSION THREAD

**Discussion by JessyCheng2003** [1101-GS4719-C/Python-Community-GS4719-C#1255] [General]
前陣子，我去參加在美IC設計工程師的講座，講者提到了在美國職場中常用到幾個英文俚語，我覺得很有趣，所以想分享給大家～

**1.Pick the low hanging fruit**
字面意思：挑最掛在低處的果實
解釋：簡單的東西能做先做，在工程師中有一個很重要的「80/20 rule」，也就是80%的工作可以在20%的時間去做完，所以把重點抓到後，80%的工作先用20%做完，剩下20%的複雜工作後面要花80%。

**2. Peel the onions** 
字面意思：剝洋蔥
解釋：抽絲剝繭，循序漸進。今天要去做一件事情時，要一步一步的往核心。例如: It's a complicated situation. We've gotta peel the onions.

**3.bite the bullet**
字面意思：咬著子彈
解釋：咬牙撐過去。有時候有些project很難做，卻非做不可，工程師就會說 We’ll bite the bullet!

**4.heads up**
字面意思：頭伸出來
解釋：預告。如果想和同事預告事情就會用到，例如 I'm giving you a heads up that I'll take a vacation then.

歡迎大家分享其他有趣的俚語，之後會再出Part 2!

**Comment by DuxtinTai** [comment]
有些英文的生活俚語真的好有趣，但也讓我深刻感受到自己英文不足之處，第一次聽到真的完全猜不到講者想表達的意思，我認為這也代表即使在托福等英文考試中取得佳績，也不能直接等同於可以快速融入辦公室文化。

接下來，我也來分享一些有趣的英文俚語:

"Bite the bullet": 這個俚語可能源自於18世紀時醫生在沒有麻醉劑的情況下，讓病人咬住銀製的子彈以減輕疼痛。

"Break a leg": 這個俚語通常用於祝福演員或表演者表演順利，其出處可能有不同說法，其中一種說法是源自於戲劇行業中的迷信，認為直接祝福演員好運反而會招來壞運，因此使用相反的措辭。

"Cat got your tongue": 這個俚語的出處可能源自於古代的鞭刑，鞭子的尾端裝有鉤狀物，用來抓住罪犯的舌頭，因此被抓住舌頭的人無法說話。

"Piece of cake": 這個俚語的出處可能源自於19世紀的美國，當時蛋糕是一種比較常見的甜點，因此 "piece of cake" 就成為表示 "容易完成" 的一種方式。

"Spill the beans": 這個俚語的出處可能源自於古代的競技場，競賽的時候會用豆子來記分，如果有人在比賽前把豆子洩漏出去，就會被視為作弊，因此 "spill the beans" 就成為表示 "洩漏秘密" 的一種方式。
